apps/website/docusaurus.config.js

```diff
@@ -50,6 +51,12 @@ const config = {
         //   src: 'img/logo.svg',
         // },
         items: [
+          {
+            type: 'doc',
+            docId: 'try-it-out',
+            position: 'left',
+            label: 'Try it out',
+          },
```

Comment:
Let's move this after "React" section as docs are more important :)

Reply:
I actually disagree though, I think having hands on code experience is better for docs... it's easier for ppl to have a good positive experience 

but this can also be personal biais

Reply:
That's fair, but if people don't know how to install the package they don't care what CSS it produces 🐱

Reply:
but should the install be on the main landing page ?

Reply:
- Yes, if we are React only
- No, if we will support something else except React

---

For JSS that has similar feature in docs it's also not first:

![image](https://user-images.githubusercontent.com/14183168/154814249-29779945-a84e-4f27-9661-8007ac569a47.png)

Reply:
alright, changed the order